code/tailer/datastripe.q

```diff
@@ -151,17 +151,26 @@ savetables:{[dir;tablename]
     .gc.run[];
     };
 
-savealltables:{[dir]
+savealltablesoverperiod:{[dir;lasttime]
```

Comment:
I can't see where `lasttime` is used, am I just being blind?

Reply:
I believe this is just a vestige of how the function used to operate, this can be safely removed